lib/pacemaker/pcmk_cluster_queries.c

```diff
@@ -361,6 +418,25 @@ pcmk__list_nodes(pcmk__output_t *out, gboolean BASH_EXPORT)
     return pcmk_legacy2rc(rc);
 }
 
+#ifdef BUILD_PUBLIC_LIBPACEMAKER
+int
+pcmk_list_nodes(xmlNodePtr *xml)
+{
```

Comment:
Let's keep this one function guarded with BUILD* until we finish the changes to crmadmin --nodes